src/components/tables/TableDataModifier.ts

```diff
@@ -79,6 +80,24 @@ export const applyFilterOnDisplayedIds = (
   return nextDisplayedIds;
 };
 
+export const applyDatePickerOnDisplayedIds = (
+  nextDisplayedIds: string[],
+  tableDataById: ITableRowData,
+  tableCompositeState: ITableCompositeState,
+  tableOwnProps: ITableOwnProps,
+): string[] => {
+  const { from, to } = tableCompositeState;
+  const { datePicker } = tableOwnProps;
+  if (from && to && datePicker && datePicker.attributeValue) {
+    nextDisplayedIds = nextDisplayedIds.filter((dataId: string): boolean => {
+      const rowDate = moment(tableDataById[dataId][datePicker.attributeValue]).toDate();
```

Comment:
I think you can use `return moment(tableDataById[dataId][datePicker.attributeValue]).isBetween(from, to);`

Reply:
I didn't know that, thanks!

Reply:
TIL